### ISSUE TITLE
7.11.9 Employee role control is lost, invalid

### ISSUE THREAD

**Issue by copyyour** [salesagility/SuiteCRM#8178]
Employee role permissions are lost and role control does not work.

In 7.11.8 I set up different employee roles and it worked fine. However, after upgrading to 7.11.9, the role is invalid and all employees have the same default permissions.

**Comment by copyyour** [comment]
There is no error message in the log.
My employee role is set to see only the person in charge in the list. But after I upgraded, the employee could see other information in the list that he was not responsible for, can see, can't edit.

**Comment by pgorod** [comment]
`Employees` is the name of a module in SuiteCRM. But from what I can understand, when you say `Employee` that is a name of a `Role` you created in your system?

**Comment by copyyour** [comment]
I am using ubuntu 18.04
Environment installation xampp
From 7.11.8 through the System Management Upgrade Wizard for a normal upgrade, no errors occurred during the period.

However, during the use process, the permission control of the user role assignment is found to be invalid. The content of the list is set to be viewable only by the person in charge. After the upgrade, it becomes a full member to view.

BTW: I don't know English well, I use google translation, sorry.

**Comment by pgorod** [comment]
Maybe this will solve your problem #8192

This should be in the next version

**Comment by copyyour** [comment]
w I modified it in the way of #8192 , but the problem still exists.

The list of roles is still visible to all employees

I'm pretty sure I'm setting up a list that only allows the responsible person to see the entries they created.

**Comment by copyyour** [comment]
<img width="1365" alt="截屏2019-11-06下午10 49 53" src="https://user-images.githubusercontent.com/27992939/68308812-7004d900-00e8-11ea-851c-391289580379.png">
The first picture is the role permission of a user I set. It is very clear that I set him to only browse the entries he created in the list.

<img width="1432" alt="截屏2019-11-06下午10 51 35" src="https://user-images.githubusercontent.com/27992939/68308959-ae9a9380-00e8-11ea-9a28-911457d6af18.png">
The second picture shows that although I set the permissions, he can still see the list that he did not belong to, although it can not be edited, but can be viewed, deleted, and exported.

So, I think the role control is invalid after upgrading to 7.11.9.
